examples/benchmark/parser.js

```diff
@@ -1,7 +1,7 @@
 'use strict'
 
 const fastify = require('../../fastify')({
-  logger: false
+  logger: true
```

Comment:
Do we need it?
It may slow down the test

Reply:
It was just for debugging, I will remove it.